src/envoy/http/mixer/report_data.h

```diff
@@ -18,12 +18,17 @@
 #include "common/request_info/utility.h"
 #include "envoy/http/header_map.h"
 #include "envoy/request_info/request_info.h"
+#include "extensions/filters/http/well_known_names.h"
 #include "include/istio/control/http/controller.h"
 #include "src/envoy/utils/utils.h"
 
 namespace Envoy {
 namespace Http {
 namespace Mixer {
+static const std::string permissive_policy_id_field =
```

Comment:
Per google c++ styple guide,  const variable should be kCamelCase. 
Please also add comment on them
If tcp rbac shares the same name,  consider move them into utils folder

Reply:
updated; perfer to move later(as mentioned in the other comment).